resolwe_bio/processes/support_processors/prepare_geo.yml

```diff
@@ -117,8 +117,8 @@
           MD5=($(md5sum "${NAME}/${SAMPLE_NAME}.wig"))
           echo -e "${SAMPLE_NAME}.wig\t${SAMPLE_NAME}\t${MD5}\t${SAMPLE_TO_PAIRED[${SAMPLE_NAME}]}" >> "${NAME}.txt"
 
-          {% if sample_id in sample_ids and sample_id not in background_ids %}
-            {{ ':' if not background_ids.append(sample_id) }}
+          {% if background_id in sample_ids and background_id not in background_ids %}
+            # {{ background_ids.append(background_id) }}
```

Comment:
Is `#` here by mistake? Also below..

Reply:
This is not a mistake. The jinja expression gets evaluated even if it's commented and that's exactly what I wanted here - append `background_id` without calling a bash command.

Reply:
I'd suggest using `echo` for such cases. For example:
```
echo Background: {{ background_ids.append(background_id) }}
```

Reply:
this will echo `Background: None` every time. I don't understand how this is better than the commented version.

Reply:
Oh - if you don't need the statement's output, just use `{% statement %}`

(if statement's output is always "None", it doesn't even help with debugging)